src/components/operations/operation-details/react/runtime/OperationConsole.tsx

```diff
@@ -133,6 +138,33 @@ export const OperationConsole = ({
         return await tenantService.getServiceSkuName();
     }
 
+    const setSoapHeaders = (): void => {
+        const representation = consoleOperation.current.request.representations?.[0];
```

Comment:
Do we need validate (log errors if this input is not expected? More than 1 element in array, for example